Makefile

```diff
@@ -39,7 +39,7 @@ build:
 
 core_api_gen:
     ifeq (, $(shell which oapi-codegen))
-	go install github.com/deepmap/oapi-codegen/cmd/oapi-codegen@latest
+	go install github.com/deepmap/oapi-codegen/cmd/oapi-codegen@v1.12.4
```

Comment:
note that if this isn't pinned, a new developer will have a newer version installed which causes minor formatting differences across all generated files

Reply:
We are actually planning to move to v2 of this library in a separate PR. Just FYI

Reply:
Hey, I have created https://github.com/astronomer/astro-cli/pull/1552, hopefully this helps new devs to always use the correct version.